js/src/common/utils/Drawer.js

```diff
@@ -41,7 +62,10 @@ export default class Drawer {
 
     const $app = $('#app');
 
-    if (!$app.hasClass('drawerOpen')) return;
+    this.focusTrap.deactivate();
+    window.removeEventListener('resize', this.throttledResizeHandler);
```

Comment:
Are we calling this onremove too? Or just on hide?

Reply:
I think only on hide should be fine. The Drawer class isn't a component, so we don't have access to the `onremove` lifecycle hook here, and, as such, this class never gets "recreated" in the same way mithril components can.